staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -900,10 +919,15 @@ func (r *Request) handleWatchList(ctx context.Context, w watch.Interface) WatchL
 				lastKey = key
 			case watch.Bookmark:
 				if meta.GetAnnotations()[metav1.InitialEventsAnnotationKey] == "true" {
+					base64EncodedInitialEventsListBlueprint := meta.GetAnnotations()[metav1.InitialEventsListBlueprintAnnotationKey]
+					if len(base64EncodedInitialEventsListBlueprint) == 0 {
+						return WatchListResult{err: fmt.Errorf("%q annotation is missing content", metav1.InitialEventsListBlueprintAnnotationKey)}
```

Comment:
i need to make sure that we have a unit test for this path.

Reply:
Nit: I expected an error case when the annotation is missing, but IIRC `""` is the valid base64 encoding of `""`. I would expect that to fail during the call to Decode.